test/Libraries/RevitIntegrationTests/FamilyTests.cs

```diff
@@ -37,6 +40,45 @@ public void GetFamilyInstancesByType()
         }
 
         [Test]
+        [TestModel(@".\Family\GetFamilyInstancesByType.rvt")]
+        public void GetFamilyInstancesByName()
+        {
+            var model = ViewModel.Model;
+
+            string samplePath = Path.Combine(workingDirectory, @".\Family\GetFamilyInstancesByName.dyn");
+            string testPath = Path.GetFullPath(samplePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
+
+            RunCurrentModel();
+
+            //assert number of loaded families in document:
```

Comment:
This comment says "assert", but there is no assert here. Not a big deal, but somewhat confusing. The same for the other comment below about asserting.

Reply:
Can we add a summery of the test somewhere, e.g. sth like `This test check to ensure that deleting the family.byName node does not delete the family in Revit host document`